NuGet/Definition/AspNetCore MVC.NuSpec

```diff
@@ -25,10 +25,17 @@ Supports the creation of AspNetCore applications.
         <dependency id="Microsoft.AspNetCore" version="1.1.1" />
         <dependency id="Microsoft.AspNetCore.Mvc" version="1.1.2" />
       </group>
+      <group targetFramework="netstandard2.0">
+        <dependency id="CSLA-Core" version="[4.6.3-Beta10]" />
```

Comment:
Just curious about the exact version match on [4.6.3-Beta10] - do you have automation that updates this with each build? Is it not preferable to allow a range?

Reply:
Yes, there's automation that updates this when the NuGet packages are built.